packages/drivers/local-driver/src/localSessionStorageDb.ts

```diff
@@ -161,15 +222,25 @@ class LocalSessionStorageCollection<T> implements ICollection<T> {
         }
     }
 
+    /**
+     * Finds the query in session storage and returns its value.
+     * Returns null if query is not found.
+     * Query is expected to have a member "_id" which is a unique id.
+     *
+     * @param query - what to find in the database
+     *
+    */
     private findOneInternal(query: any): any {
         if (query._id) {
-            const json = sessionStorage.getItem(`${this.collectionName}-${query._id}`);
+            const json = sessionStorage.getItem(`${this.collectionName}-${query._id}`);// form of data
             if (json) {
                 // eslint-disable-next-line @typescript-eslint/no-unsafe-return
                 return JSON.parse(json);
             }
         } else {
             const queryKeys = Object.keys(query);
+
+            // Optimization needed
```

Comment:
```suggestion
```

Either create an issue and reference the issue number here or remove comment (currently unclear what work is needed)